--- ../master/disk_usage_original.py	2019-12-18 23:25:53.526470647 +0000
+++ disk_usage_fixed.py	2019-12-18 23:30:46.283282074 +0000
@@ -1,6 +1,7 @@
 #!/usr/bin/#!/usr/bin/env python3
 
 import shutil
+import sys
 
 def check_disk_usage(disk, min_absolute, min_percent):
     """Returns True if there is enough free disk space, false otherwise."""
@@ -14,9 +15,9 @@
     return True
 
 # Check for at least 2 GB and 10% free
-if not check_disk_usage("/", 2*2**30, 10):
+if not check_disk_usage("/", 2, 10):
     print("ERROR: Not enough disk space")
-    return 1
+    sys.exit(1)
 
 print("Everything ok")
-return 0
+sys.exit(0)
